hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4142,38 +4141,60 @@ func (r *HostedClusterReconciler) validateReleaseImage(ctx context.Context, hc *
 		currentVersion = &version
 	}
 
-	// Validate release image is multi-arch
-	if hc.Spec.Platform.Type == hyperv1.AWSPlatform && hc.Spec.Platform.AWS.MultiArch {
-		isMultiArchReleaseImage, err := registryclient.IsMultiArchManifestList(ctx, hc.Spec.Release.Image, pullSecretBytes)
-		if err != nil {
-			return fmt.Errorf("failed to determine if release image multi-arch: %w", err)
-		}
+	minSupportedVersion := supportedversion.GetMinSupportedVersion(hc)
 
-		if !isMultiArchReleaseImage {
-			return fmt.Errorf("release image is not a multi-arch image")
-		}
+	return supportedversion.IsValidReleaseVersion(&version, currentVersion, &supportedversion.LatestSupportedVersion, &minSupportedVersion, hc.Spec.Networking.NetworkType, hc.Spec.Platform.Type)
+}
+
+// determineHostedClusterPayloadType returns the HostedCluster payload's CPU architecture type
```

Comment:
This uses PayloadType / CPU architecture and sets it to .payloadArch
can we please use a single term consistently across the base code?

Reply:
Updated